lib/ansible/modules/database/postgresql/postgresql_ext.py

```diff
@@ -111,12 +120,20 @@ def ext_delete(cursor, ext):
     else:
         return False
 
-def ext_create(cursor, ext):
+def ext_create(cursor, ext, ver=None):
     if not ext_exists(cursor, ext):
-        query = 'CREATE EXTENSION "%s"' % ext
+        if ver:
+            query = 'CREATE EXTENSION "%s" VERSION "%s"' % ( ext, ver)
```

Comment:
Don't use string parameters interpolation (`%`) to pass variables to a SQL query string ([see](http://initd.org/psycopg/docs/usage.html#the-problem-with-the-query-parameters)). Same comment applies to queries below.